Tests/SonarScanner.MSBuild.PreProcessor.Test/Infrastructure/MockAnalyzerInstaller.cs

```diff
@@ -29,6 +30,14 @@ namespace SonarScanner.MSBuild.PreProcessor.Test
 {
     internal class MockAnalyzerInstaller : IAnalyzerInstaller
     {
+        #region Constructors
+        public MockAnalyzerInstaller() { }
+
+        public MockAnalyzerInstaller(string localCacheDirectory) {
```

Comment:
pedantic: to follow the convention from the rest of the code block should start on the next line.